.buildkite/filebeat/filebeat-pipeline.yml

```diff
@@ -61,7 +68,9 @@ steps:
 
   - label: ":linux: Load dynamic filebeat pipeline"
     key: "filebeat-pipeline"
-    command: ".buildkite/filebeat/generate_filebeat_pipeline.sh"
+    command: ".buildkite/scripts/generate_filebeat_pipeline.sh"
```

Comment:
Should be: command: ".buildkite/filebeat/generate_filebeat_pipeline.sh"

Reply:
no, it should be the same as it is in the main